packages/jest/src/jest.builder.ts

```diff
@@ -5,21 +5,31 @@ import {JestBuilderSchema} from './schema';
 
 import {OptionsConverter} from "./options-converter";
 import {JestConfigurationBuilder} from "./jest-configuration-builder";
+import { DefaultConfigResolver } from './default-config.resolver';
+import { CustomConfigResolver } from './custom-config.resolver';
 
 const jest = require('jest');
 
 export default class JestBuilder implements Builder<JestBuilderSchema> {
+
+  private jestConfigurationBuilder: JestConfigurationBuilder;
+  private optionsConverter: OptionsConverter;
+
   constructor(private context: BuilderContext) {
+    this.jestConfigurationBuilder = new JestConfigurationBuilder(
+      new DefaultConfigResolver(),
+      new CustomConfigResolver(context.logger)
```

Comment:
created object instances for these in favor of static utility method holders - this allowed me to provide the resolver with dependencies like the logger